src/components/dialogs/create-study-dialog.js

```diff
@@ -172,7 +172,9 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
     const [isUploadingFileInProgress, setUploadingFileInProgress] =
         useState(false);
 
-    const [fileCheckedCase, setFileCheckedCase] = useState(false);
+    const [fileCheckedCase, setFileCheckedCase] = useState(
+        providedCase ? true : false
```

Comment:
`providedCase` is always undefined because it's not passed with the parameters when this component is called `<CreateStudyForm open={true} onClose={handleCloseDialog} />`

Reply:
Are you sure ? On main CreateStudyDialog is instanciated this way in `content-contextual-menu.js` :
`<CreateStudyDialog
  open={true}
  onClose={handleCloseDialog}
  providedCase={activeElement}
/>`

Reply:
you're right. You can replace this line by const [fileCheckedCase, setFileCheckedCase] = useState(!!providedCase);